dashboard/dashboard-statement-details.md

```
@@ -21,7 +21,7 @@ TiDB v6.6.0以降、高速プランバインディング機能が導入されま
 
 #### 実行計画をバインドする {#bind-an-execution-plan}
 
-1.  **「プランのバインド」**をクリックします。「**プランのバインド」**ダイアログボックスが表示されます。
+1.  **プランのバインド**をクリックします。**プランのバインド**ダイアログボックスが表示されます。
```

markdownlint-cli2 (0.23.2)

[warning] Spaces after list markers
Expected: 1; Actual: 2

(MD030, list-marker-space)